percona-xtradb-cluster-5.7/dockerdir/etc/mysql/node.cnf

```diff
@@ -20,6 +20,9 @@ wsrep_slave_threads=2
 wsrep_cluster_address=gcomm://
 wsrep_provider=/usr/lib64/galera3/libgalera_smm.so
 
+#extra_max_connections=1
+#extra_port=33062
+
```

Comment:
we should add these lines similar to `wsrep_node_incoming_address ` https://github.com/percona/percona-xtradb-cluster-operator/blob/master/build/pxc-configure-pxc.sh#L66-L67

reason - old releases should work with new operator